Sources/ASN1Kit/encodable/GeneralizedTimeDateFormatter.swift

```diff
@@ -55,23 +54,19 @@ public class GeneralizedTimeDateFormatter {
 // swiftlint:disable strict_fileprivate
 
 extension Date {
+    // If `from` was ""19851106210627.3-0500" then ".3" should be added to self
     fileprivate func byAddingFraction(from date: String) -> Date {
```

Comment:
Maybe worth adding a test for this method?

Reply:
You're right. It's implicitly (and extensively) tested via `DateExtASN1EncodingTest`